client/request.go

```diff
@@ -158,17 +158,21 @@ func (p *pair) Less(i, j int) bool {
 func (r *Request) Headers() iter.Seq2[string, []string] {
 	return func(yield func(string, []string) bool) {
 		peekKeys := r.header.PeekKeys()
-		keys := make([][]byte, len(peekKeys))
-		copy(keys, peekKeys) // It is necessary to have immutable byte slice.
+
+		// It is necessary to have immutable byte slice.
```

Comment:
@arturmelanchyk the comment is a little bit confusing, since you create a byte slice with mutable values out of the key


```suggestion
```